src/routes/admin.js

```diff
@@ -1,4 +1,7 @@
 import express from 'express';
+/* eslint-disable import/no-unresolved, node/no-missing-import */
+import { stringify as csvStringify } from 'csv-stringify/sync'; // https://github.com/adaltas/node-csv/issues/323
```

Comment:
These eslint issues seem like a known issue. I tried using the `csv` package instead, but that didn't work either :disappointed:

Reply:
I had a similar issue with `string-strip-html` [here](https://github.com/ckolderup/postmarks/blame/4c6efe019b9f80bc4be46fb6bbe8a5225bef0da5/src/bookmarks-db.js#L12)-- if anyone reading this knows why this happens, I'd love to know!